app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -0,0 +1,270 @@
+import { createContext, useContext, useEffect, useState } from 'react';
+import type { ReactNode } from 'react';
+import useObjectMemo from '@18f/identity-react-hooks/use-object-memo';
+import DeviceContext from './device';
+import AnalyticsContext from './analytics';
+
+/** Global declarations */
+declare let AcuantJavascriptWebSdk: AcuantJavascriptWebSdkInterface; // As of 11.7.0, this is now a global object that is not on the window object.
+declare let AcuantCamera: AcuantCameraInterface;
+
+/**
+ * @see https://github.com/Acuant/JavascriptWebSDKV11/blob/11.4.3/SimpleHTMLApp/webSdk/dist/AcuantJavascriptWebSdk.js#L1025-L1027
+ * @see https://github.com/Acuant/JavascriptWebSDKV11/blob/11.4.3/SimpleHTMLApp/webSdk/dist/AcuantJavascriptWebSdk.js#L1049
+ */
+interface AcuantConfig {
+  path: string;
+}
+
+interface AcuantCameraInterface {
+  isCameraSupported: boolean;
+}
+/**
+ * @see https://github.com/Acuant/JavascriptWebSDKV11/blob/11.4.4/SimpleHTMLApp/webSdk/dist/AcuantJavascriptWebSdk.js#L1327-L1353
+ */
+type AcuantInitializeCode = 1 | 2 | 400 | 401 | 403;
+
+interface AcuantCallbackOptions {
+  onSuccess: () => void;
+  onFail: (code: AcuantInitializeCode, description: string) => void;
+}
+
+type AcuantInitialize = (
+  credentials: string | null,
+  endpoint: string | null,
+  callbackOptions?: AcuantCallbackOptions,
+) => void;
+
+type AcuantWorkersInitialize = (callback: () => void) => void;
+
+declare global {
+  interface AcuantJavascriptWebSdkInterface {
+    initialize: AcuantInitialize;
+    startWorkers: AcuantWorkersInitialize;
+    START_FAIL_CODE: string;
+    REPEAT_FAIL_CODE: string;
+    SEQUENCE_BREAK_CODE: string;
+  }
+}
+
+/**
+ * Start liveness capture
+ */
+type AcuantStartSelfieCapture = (callback: (nextImageData: string) => void) => void;
+interface AcuantPassiveLivenessInterface {
+  startSelfieCapture: AcuantStartSelfieCapture;
+}
+
+declare global {
+  interface Window {
+    /* Document load callback to assign Javascript Web SDK globals */
+    loadAcuantSdk: () => void;
+    /* Acuant configuration */
+    acuantConfig: AcuantConfig;
+    /* Acuant Passive Liveness API */
+    AcuantPassiveLiveness: AcuantPassiveLivenessInterface;
+    /* Possible AcuantJavascriptWebSdk on the window object (11.5.0) */
+    AcuantJavascriptWebSdk: AcuantJavascriptWebSdkInterface;
+    /* Possible AcuantCamera on the window object (11.5.0) */
+    AcuantCamera: AcuantCameraInterface;
+  }
+}
+
+interface AcuantContextProviderProps {
+  sdkSrc: string; // SDK source URL.
```

Comment:
Would it also be helpful to convert other kinds of one-liner comment blocks that are describing reasoning in the code, or do you only want docblock for attribute and function comments? [Here](https://github.com/18F/identity-idp/blob/eric-acuant-sdk-update/app/javascript/packages/document-capture/context/acuant.tsx#L177) is an example of what I mean by the former.
  
(Am not a VSCode user, so I don't see it)

Reply:
I'm not as concerned about those, and in fact may prefer to keep it as-is, since that's more describing the general flow of logic than any one single thing.

Reply:
Updated in b563c26